use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,227 @@
+               ],
+               "Resource": "*"
+           }
+       ]
+   }
+   ```
+
+
+
+### How to use ECS Task Metadata in Fluent Bit config?
+
+The init process injects ECS Task Metadata into the Fluent Bit container as environment variables:
+
+```
+AWS_REGION / ECS_LAUNCH_TYPE / ECS_CLUSTER / ECS_FAMILY
+ECS_TASK_ARN / ECS_TASK_ID / ECS_REVISION / ECS_TASK_DEFINITION
+```
+
+You can use them as env vars directly in the Fluent Bit config.
+
+**Example:**
+
+```
+[OUTPUT]
+    Name   		cloudwatch_logs
+    Match  		*
+    region 		${AWS_REGION}
+```
+
+
+
+### How init process works?
+
+1. The init process will request [ECS Task Metadata](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v4.html). Get valuable parts from the responses and set them using `export` as environment variables.
+
+2. After you set the ARN or Path of the config files as environment variables in the ECS Task Definition, the init process will collect all config files which you specified (download files which come from S3 if needed).
+
+3. The init process will process these config files one by one, use @INCLUDE keyword to add config files to the main config file. And the init process will check if each config file is a parser config. If it is a parser config, change the original Fluent Bit command, add -R to specify that parser. For more details, please see the example below.
+
+4. The init process will finally invoke Fluent Bit with the modified main configuration file.
+
+
+
+**Example:**
+
+Suppose you specify config files as environment variable in the FireLens configuration as:
+
+```
+"environment": [
+                {
+                    "name": "aws_fluent_bit_init_s3_1",
+                    "value": "arn:aws:s3:::yourBucket/your-filter.conf"
+                },
+                {
+                    "name": "aws_fluent_bit_init_s3_2",
+                    "value": "arn:aws:s3:::yourBucket/your-parser.conf"
+                },
+                {
+                    "name": "aws_fluent_bit_init_s3_3",
+                    "value": "arn:aws:s3:::yourBucket/your-s3-output.conf"
+                }
+            ]
+```
+
+**your-filter.conf** (new config you created and init process downloaded from S3, path:`/init/fluent-bit-init-s3-files/your-filter.conf`)
+
+```
+[FILTER]
+    Name          parser
+    Match         xxxx
+    Key_Name      data
+    Parser        example
+```
+
+**your-parser.conf** (new config you created and init process downloaded from S3, path:`/init/fluent-bit-init-s3-files/your-parser.conf`)
+
+```
+[PARSER]
+    Name          example
+    Format        regex
+    Regex         ^(?<INT>[^ ]+) (?<FLOAT>[^ ]+) (?<BOOL>[^ ]+) (?<STRING>.+)$
+```
+
+**your-s3-output.conf** (new config you created and init process downloaded from S3, path:`/init/fluent-bit-init-s3-files/your-s3-output.conf`)
+
+```
+[OUTPUT]
+    Name                         s3
+    Match                        xxxx
```

Comment:
why not have all of the `Match` patterns jsut be `*` in the example? its more typical... the `xxx` might confuse folks I fear